src/gradleinttest.sh

```diff
@@ -21,12 +21,16 @@ cleaniampolicy () {
 }
 
 gradleinttest () {
-  if [ -f env_vars ] && [[ -n "${IT_JADE_API_URL}" ]] && [[ "${test_to_run}" == "testIntegration" ]]; then
-    echo "Getting GCR tags and IT_JADE_API_URL for integration test"
+  if [ -f env_vars ] && [[ -n "${IT_JADE_API_URL}" ]]; then
+    echo "Getting GCR tags and IT_JADE_API_URL for connected or integration test"
     eval $(cat env_vars)
-  else
-    echo "Skipping importing environment vars for gradleinttest"
   fi
+
+  # determine id of integration project
+  namespace_number=$(echo ${NAMESPACEINUSE} | sed 's/integration-//g')
```

Comment:
I think this won't be defined in the connected test case, is this correct? If so, the GOOGLE_CLOUD_DATA_PROJECT will also be invalid, right?

Reply:
Here is where the `${NAMESPACEINUSE}` gets defined: https://github.com/broadinstitute/datarepo-actions/blob/3c85a3606e8b0ac5805c559cc8ab185e7081eef8/src/checknamespace.sh#L19

Reply:
Yes, agreed.